File: 200B - Drinks.cpp
#include <iostream>
 
using namespace std;
 
int main()
{
    double n,sum=0,x;
    cin>>n;
    for(int i =0 ;i<n;i++){
        cin>>x;
        sum=sum+x;
    }
    cout<<sum/n;
 
    return 0;
}

File: 200B - Drinks.java
import java.util.Scanner;
 
public class Drinks {
 
    public static void main(String[] args) { 
        Scanner sc =  new Scanner (System.in);
        int x = sc.nextInt();
        int []y = new int [x];
        double sum = 0 ; 
        for (int i = 0 ; i < x ; i ++){
        
            y[i]=sc.nextInt();
            sum=sum+y[i];
        }
        System.out.println(sum/x);
    }
    
}